src/src/Form/AbandonedCartFormType.php

```diff
@@ -0,0 +1,223 @@
+                )->add(
+                    'name',
+                    CheckboxType::class,
+                    [
+                        'label' => $this->trans('Product name', 'Modules.Smailyforprestashop.Admin'),
+                        'required' => false,
+                        'attr' => [
+                            'material_design' => true,
+                        ],
+                    ]
+                )->add(
+                    'description',
+                    CheckboxType::class,
+                    [
+                        'label' => $this->trans('Product description', 'Modules.Smailyforprestashop.Admin'),
+                        'required' => false,
+                        'attr' => [
+                            'material_design' => true,
+                        ],
+                    ]
+                )->add(
+                    'sku',
+                    CheckboxType::class,
+                    [
+                        'label' => $this->trans('Product SKU', 'Modules.Smailyforprestashop.Admin'),
+                        'required' => false,
+                        'attr' => [
+                            'material_design' => true,
+                        ],
+                    ]
+                )->add(
+                    'price',
+                    CheckboxType::class,
+                    [
+                        'label' => $this->trans('Product price', 'Modules.Smailyforprestashop.Admin'),
+                        'required' => false,
+                        'attr' => [
+                            'material_design' => true,
+                        ],
+                    ]
+                )->add(
+                    'base_price',
+                    CheckboxType::class,
+                    [
+                        'label' => $this->trans('Product base price', 'Modules.Smailyforprestashop.Admin'),
+                        'required' => false,
+                        'attr' => [
+                            'material_design' => true,
+                        ],
+                    ]
+                )->add(
+                    'quantity',
+                    CheckboxType::class,
+                    [
+                        'label' => $this->trans('Product quantity', 'Modules.Smailyforprestashop.Admin'),
+                        'required' => false,
+                        'attr' => [
+                            'material_design' => true,
+                        ],
+                    ]
+                )
+            )
+            ->add('sync_interval', NumberType::class, [
+                'label' => $this->trans('Abandoned Cart Delay', 'Modules.Smailyforprestashop.Admin'),
+                'html5' => true,
+                'attr' => [
+                    'step' => 1,
+                ],
+                'help' => $this->trans('Time after cart is considered abandoned after last cart edit from customer. Minimum 15 minutes.', 'Modules.Smailyforprestashop.Admin'),
```

Comment:
I looked into it and figured out that this is a PrestaShop-s templating engine that occasionally displays those messages twice. In version 8.0.5 or was it 8.0 they display all help messages twice. So 8.1.5 has some improvement in that case. A solution would be to implement those help texts directly in templates but I think this will make the code harder to maintain.

Reply:
Agreed. It would make the code harder to maintain. Assuming this is a PrestaShop bug and marking this question resolved.